tests/HashMapProperties.hs

```diff
@@ -225,6 +225,45 @@ pAlterFLookup k f =
   `eq`
   getConst . HM.alterF (Const . apply f) k
 
+pSubmapReflexive :: [(Key, Int)] -> Bool
+pSubmapReflexive xs =
+  let m = HM.fromList xs
```

Comment:
I tried to switch to `HashMap Key Int`, however, there is no `Arbitrary` instance yet. Should I create one for now until #154 is resolved?

Reply:
Oh, apparently the instance is in `tests/Strictness.hs`. Maybe it could be shared? I'm not very familiar with the structure of the testsuites yet.

Duplicating it here seems good enough too, though.

Or leave it for future work. It's not really your responsibility IMHO.